src/hooks/useActiveKeyPress.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from 'react';
+
+const useActiveElementKeypress = (focusId, targetKey) => {
+  // State for keeping track of whether key is pressed
+  const [keyPressed, setKeyPressed] = useState(false);
+  useEffect(() => {
+    // If pressed key is our target key then set to true
+    function downHandler(e) {
+      const { key } = e;
+      if (key === targetKey) {
+        e.preventDefault();
+        setKeyPressed(true);
+      }
+    }
+
+    // If released key is our target key then set to false
+    const upHandler = ({ key }) => {
+      if (key === targetKey) {
+        setKeyPressed(false);
+      }
+    };
+    const elem = document.getElementById(focusId);
```

Comment:
The use of an id to get a reference to an element seems problematic in the context of React. We should rework this to use `ref` (https://reactjs.org/docs/refs-and-the-dom.html) if we need to attach event handlers manually like this.

That said, this work is best not abstracted into a hook. React will help us attach an remove event listeners. Let's rely on React to do the work. I'll make an alternative suggestion in Interactive button.